content/docs/user-guide/experiment-management/visualizing-plots.md

```diff
@@ -154,9 +154,10 @@ top-level definition, DVC will create separate rendering for each type.
 Plots can be defined in a top-level `plots` key in `dvc.yaml`. Top-level plots
 can use any file found in the <abbr>project</abbr>.
 
-In the simplest use, you only need to provide the plot's file path. In the
-example below, DVC will take data from `logs.csv` and use the default plotting
-behavior (apply the `linear` plot [template] to the last found column):
+In the simplest use, you only need to provide the plot's file path as a
+dictionary key. In the example below, DVC will take data from `logs.csv` and use
```

Comment:
Looks like we forgot to update the sample around line 25?

<img width="1206" alt="image" src="https://user-images.githubusercontent.com/1477535/198204375-d6e92dc8-5a53-45e1-9141-dc1f9d560f59.png">

Reply:
May also be a good opportunity to address https://github.com/iterative/dvc.org/pull/4072#pullrequestreview-1155961401 (up to you).

Reply:
Good points, @jorgeorpinel! Looking at https://github.com/iterative/dvc.org/pull/4072#discussion_r1005296355 again, I'm going to keep it since I think it's intentional and serves a purpose. It's a quick way to show how you combine everything from below. It might make more sense somewhere further down if we had some kind of conclusion section, but for now I would rather have it than not.

Reply:
Kk